requirements/extra.txt

```diff
@@ -5,6 +5,6 @@ horovod>=0.21.2,!=0.24.0  # no need to install with [pytorch] as pytorch is alre
 torchtext>=0.9.*
 omegaconf>=2.0.5
 hydra-core>=1.0.5
-jsonargparse[signatures]>=4.5.0
+jsonargparse[signatures]>=4.6.0.dev3
```

Comment:
let's wait it is a stable release, not a RC or even dev version

Reply:
jsonargparse 4.6.0 is now released.